Samples/android/Kotlin/Android-SDK-Demo-App/app-v7-java/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    namespace 'com.vungle.samples.samplejava'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.vungle.samples.samplejava"
+        minSdk 19
+        targetSdk 33
+        versionCode 1
+        versionName "1.0"
+        multiDexEnabled true
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    buildFeatures {
+        viewBinding true
+    }
+}
+
+dependencies {
+    implementation 'androidx.core:core-ktx:1.8.0'
+    implementation 'androidx.appcompat:appcompat:1.5.1'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation 'androidx.navigation:navigation-fragment-ktx:2.5.3'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.5.3'
+    implementation 'androidx.recyclerview:recyclerview:1.2.1'
+    implementation 'com.google.android.material:material:1.6.1'
+    implementation 'androidx.leanback:leanback:1.0.0'
+    testImplementation 'junit:junit:4.13.2'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.4'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.0'
+//    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.4.1")
+    implementation 'com.android.support:multidex:1.0.3'
+
+    implementation 'com.github.vungle:vng-android-sdk:7.0.0-early1-RC8'
+
+    // Vungle SDK dependencies
+//    implementation files('libs/vungle-7.0.0-early1-RC7.aar')
+//    implementation files('libs/vungle-7.0.0-early1-RC7.jar')
+//    implementation files('libs/kotlinx-serialization-json-jvm-1.4.1.jar')
+//    implementation files('libs/okhttp-4.10.0.jar')
+//    implementation files('libs/okio-jvm-2.8.0.jar')
+//    implementation files('libs/protobuf-kotlin-3.21.7.jar')
```

Comment:
Need to update with RC8 dependencies before releasing.

Reply:
update to EA1